docs/guide/install-cmake.rst

```diff
@@ -0,0 +1,99 @@
+.. Copyright 2003-2022 by Wilson Snyder.
+.. SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+.. _Installation:
```

Comment:
```suggestion
.. _CMakeInstallation:
```

Also you need to add this file to the document in index.rst

Please also do a "make spelling" and check if any issues, and "make info" in top makefile and look at PDF to see if this looks good and doesn't get any warnings when builds, thanks.

Reply:
I am currently unable to run the spelling check
latex is getting generated